<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>training_window</class>
 <widget class="QMainWindow" name="training_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="training_lbl_here">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Here is the place where everything begins</string>
    </property>
   </widget>
   <widget class="QLabel" name="training_lbl_trainings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trainings</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="training_gb_new_training">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>761</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>New training</string>
    </property>
    <widget class="QPushButton" name="training_btn_start">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="training_lbl_text">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QComboBox" name="training_cmb_readings">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Users previously logged into the system</string>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="training_gb_previous_training">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Your previous trainings</string>
    </property>
    <widget class="QTableView" name="training_tbl_previous_trainings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>741</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="training_btn_logout">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="training_btn_stats">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
